include/alpaka/exec/ExecCpuOmp2Threads.hpp

```diff
@@ -173,14 +173,15 @@ namespace alpaka
                         // Therefore we use 'omp parallel' with the specified number of threads in a block.
                         #pragma omp parallel num_threads(iBlockThreadCount)
                         {
-                            // The first thread does some checks in the first block executed.
-                            if((::omp_get_thread_num() == 0) && (acc.m_gridBlockIdx.sum() == static_cast<TIdx>(0)))
+                            #pragma omp single nowait
                             {
-                                int const numThreads(::omp_get_num_threads());
-                                alpaka::ignore_unused(numThreads);
-#if ALPAKA_DEBUG >= ALPAKA_DEBUG_MINIMAL
-                                std::cout << BOOST_CURRENT_FUNCTION << " omp_get_num_threads: " << numThreads << std::endl;
-#endif
+                                // The OpenMP runtime does not create a parallel region when only one thread is required in the num_threads clause.
```

Comment:
Since now it seems this is not what causes the issue, maybe rewrite this comment?

Reply:
No, this is not the Windows issue.
The linux CI showed that the OpenMP runtime does not necessarily create a parallel region when only one thread is requested.

Reply:
Ah. I got really confused by your statement 'This shows that OpenMP on windows does not work at all right now.'